homeworks/homework-1/app/src/components/common/Header.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import {
+  Image,
+  PageHeader
+} from "react-bootstrap";
+import PropTypes from 'prop-types';
+import logo from "../../logo.jpg";
+import './Header.css';
+
+class Header extends React.Component {
+
+  render() {
+    return (
+      <PageHeader className="header">
+        <Image src={logo} className="header__logo"/>{this.props.pageTitle}
+      </PageHeader>
+    );
+  }
+}
+
+Header.propTypes = {
```

Comment:
please specify `proptypes` and `defaultprops` as static getter in class at first lines of component for example:
```
export class Component extends React.Component {
  static get propTypes() {
    return {
      ...
    };
  }
  
  static get defaultProps() {
    return {
      ...
    }
  }
```